packages/expo-modules-core/ios/Swift/Modules/Uuid/Uuidv5.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
```

Comment:
Like on Android, let's move this file out of the `Modules` directory (`Uuidv5.swift` is not a module). It could be `ios/Swift/UUIDv5.swift`